src/main/java/cn/nukkit/level/light/SkyLightUpdate.java

```diff
@@ -0,0 +1,21 @@
+package cn.nukkit.level.light;
+
+import cn.nukkit.level.ChunkManager;
+
+/**
+ * author: dktapps
```

Comment:
@Creeperface01 It's other language of programming. You can delete that. ;)
You are author now because you "created". ;)